packages/evm-adapter/src/getEthPaymentTx.ts

```diff
@@ -24,7 +24,6 @@ export const getEthPaymentTx = async (
       value: BigNumber.from(req.amount),
       gasLimit,
       gasPrice: await provider.getGasPrice(),
-      nonce: await provider.getTransactionCount(req.walletAddress),
```

Comment:
is there a diff in MM and phantom for this nonce being empty?

Reply:
Yes, MM ignores whatever we put in the nonce, and for phantom we have to find out nonce. This one is behind, we have to do it before signing..